lib/wise_homex/json_parser.ex

```diff
@@ -6,47 +6,47 @@ defmodule WiseHomex.JSONParser do
   alias Ecto.Changeset
 
   @structs %{
-    "account-email-settings" => WiseHomex.Account.EmailSettings,
-    "account-users" => WiseHomex.AccountUser,
-    "accounts" => WiseHomex.Account,
-    "addresses" => WiseHomex.Address,
-    "admin-integrations" => WiseHomex.AdminIntegration,
-    "angel-notes" => WiseHomex.AngelNote,
-    "configurable-meter-id" => WiseHomex.ConfigurableMeterID,
-    "configurable-meters" => WiseHomex.ConfigurableMeters,
-    "device-authorizations" => WiseHomex.DeviceAuthorization,
-    "device-imports" => WiseHomex.DeviceImport,
-    "device-types" => WiseHomex.DeviceType,
-    "devices" => WiseHomex.Device,
-    "encryption-keys" => WiseHomex.EncKey,
-    "expenses" => WiseHomex.Expense,
```

Comment:
Only thing changed here is the indentation, and that `expenses` was removed since it is now autogenerated.